src/Game.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+
+/**
+ * Created by vasantia on 6/21/16.
+ */
+public class Game {
+
+    private Player player1;
+    private Player player2;
+    private Player currentPlayer = null;
+    private Scanner scanner = new Scanner(System.in);
+
+    public Game(Player player1, Player player2) {
+        this.player1 = player1;
```

Comment:
A simple shortcut here can be `this.player1 = this.current = player1`.  That will do the same as the two lines.